src/Microsoft.CodeQuality.Analyzers/UnitTests/ApiDesignGuidelines/DoNotDirectlyAwaitATaskTests.cs

```diff
@@ -380,25 +382,23 @@ public async Task M()
         await t;
     }
 }
-";
-            var compilationOptions = new CSharpCompilationOptions(OutputKind.DynamicallyLinkedLibrary);
-
-            // Verify diagnostic with no editor config options.
-            VerifyCSharp(code, compilationOptions, parseOptions: null, expected: GetCSharpResultAt(9, 15));
-
-            // Verify no diagnostic with editor config options that exclude OutputKind.DynamicallyLinkedLibrary.
-            var editorConfigText = "dotnet_code_quality.output_kind = ConsoleApplication";
-            VerifyCSharp(code, GetEditorConfigAdditionalFile(editorConfigText), compilationOptions, parseOptions: null);
-
-            editorConfigText = "dotnet_code_quality.CA2007.output_kind = ConsoleApplication, WindowsApplication";
-            VerifyCSharp(code, GetEditorConfigAdditionalFile(editorConfigText), compilationOptions, parseOptions: null);
+"
+                    },
+                    AdditionalFiles = { (".editorconfig", editorConfigText) }
+                },
+                SolutionTransforms =
+                {
+                    (solution, projectId) =>
+                        solution.WithProjectCompilationOptions(projectId, new CSharpCompilationOptions(OutputKind.DynamicallyLinkedLibrary))
```

Comment:
📝 This is already the default, so there is no need to specify a solution transform for it